packages/paste-core/components/pagination/src/Pagination.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import type {PaginationProps} from './types';
+import {PaginationPropTypes} from './proptypes';
+
+const Pagination = React.forwardRef<HTMLElement, PaginationProps>(({children, label, ...props}, ref) => {
+  return (
+    <Box {...safelySpreadBoxProps(props)} ref={ref} aria-label={label} as="nav" role="navigation">
+      {children}
+    </Box>
+  );
+});
+
+Pagination.displayName = 'Pagination';
+
+Pagination.propTypes = PaginationPropTypes;
```

Comment:
Just a lil curious why you opted to import these rather than define them inline?

Reply:
Since I'm sharing the `PaginationArrowButtonPropTypes` between the different `PaginationArrow` variants, I felt it was best to put all of the PropTypes in one place.